### PR TITLE
Login / Logout for user

### LINKED ISSUE #4: Users can create new Account

### PR DESCRIPTION
**Pull request by IssacL891**
Fixes #4, #6, and #5

### CODE REVIEW

File: CLI/src/main/java/com/CSCI32006/CLI/SetupDatabase.java [hunk 1]
```diff
@@ -41,39 +41,30 @@ private void setDatabaseConn(String username, String password){
...
     }
 
-    @Command(command = "logout", description = "Logs out of the database")
+    @Command(command = "logout database", description = "Logs out of the database")
     void logout() throws SQLException {
         if(jdbcTemplate != null){
             Objects.requireNonNull(jdbcTemplate.getDataSource()).getConnection().close();
             dataSource.getConnection().close();
             session.disconnect();
+            getTerminal().writer().println("Successfully disconnected!");
```
Comment (Yiroh) [inline CLI/src/main/java/com/CSCI32006/CLI/SetupDatabase.java:56]: I'm not seeing this message in terminal.

Comment (Yiroh) [reply CLI/src/main/java/com/CSCI32006/CLI/SetupDatabase.java:56]: I see this now, I didn't have the updated build like we discussed in discord, thought this was the same as the setup build.

**Comment by Yiroh** [comment]
#35 Is an issue with this as we discussed.

**Comment by Yiroh** [comment]
#36 is also an issue, full reports of the issue (stacktrace and logs) included in the issue report.

**Review by dbo8671 — APPROVED** [review]
lgtm

**Pull request closed without merging**